docs/admin/config.rst

```diff
@@ -1489,9 +1489,8 @@ Example:
 REGISTRATION_CAPTCHA
 --------------------
 
-A value of either ``True`` or ``False`` indicating whether registration of new
-accounts is protected by CAPTCHA. This setting is optional, and a default of
-``True`` will be assumed if it is not supplied.
+An optional value of either ``True`` or ``False`` indicating whether registration of new
+accounts is protected by a CAPTCHA. A default of ``True`` is assumed if not supplied.
```

Comment:
Nearly all settings are optional, so this should not be mentioned for each of them.

```suggestion
Whether registration of new accounts is protected by a CAPTCHA. Defaults to enabled.
```